std_msgs/Header header

# IMU MESSAGES
sensor_msgs/Imu imu
sensor_msgs/MagneticField magField

# GPS MESSAGES
std_msgs/Header Header
float64 Latitude
string  LatitudeRaw
float64 Longitude
string  LongitudeRaw
float64 Altitude
float64 UTMEasting
float64 UTMNorthing
int32   Zone
uint8   ZoneLetter
int32   Quality
int32   NumSatellites
